src/Core/src/HotReload/HotReloadExtensions.cs

```diff
@@ -36,6 +37,7 @@ public static void CheckHandlers(this IView? view)
 
 		public static List<MethodInfo> GetOnHotReloadMethods(this Type type) => getOnHotReloadMethods(type).Distinct(new ReflectionMethodComparer()).ToList();
 
+		[UnconditionalSuppressMessage("Trimming", "IL2080", Justification = TrimmerHelper.HotReload)]
```

Comment:
Curious about this.
I agree that it makes sense, hotreload doesn't work on trimmed apps, so that's fine.
That said, if everything works as expected I would expect the hot-reload code to be removed from the published app (as there should be no references to it). In which case it should not generate warnings.

It might be worth trying to figure out why is the code present in the published app in the first place.

Reply:
When I was getting the full list of warnings, I was commenting out `[assembly: AssemblyMetadata ("IsTrimmable", "True")]` from `Microsoft.Maui.dll`. This let me find issues in random types that might not be used in the sample app.

Let me double-check if I can just revert all these and if I see warnings in normal MAUI apps.